src/module.ts

```diff
@@ -43,14 +44,20 @@ export default defineNuxtModule<ModuleOptions>({
       'export { components as default,  components }'
     ]
 
+    nuxt.hook('components:dirs', (dirs) => {
+      componentDirs = dirs
```

Comment:
```suggestion
      componentDirs = [
        ...dirs,
        { path: resolveModule('nuxt').replace('/index.mjs', '/app') },
        { path: resolveModule('@nuxt/ui-templates').replace('/index.mjs', '/templates') }
      ]
```

This will add `nuxt` and `@nuxt/ui-templates` dirs